tests/integration/modules/molecule/asset/playbook.yml

```diff
@@ -26,7 +26,13 @@
       register: result
 
     - assert:
-        that: result is changed
+        that:
+          - result is changed
+          - result.object.url == 'https://assets.bonsai.sensu.io/68546e739d96fd695655b77b35b5aabfbabeb056/sensu-plugins-cpu-checks_4.0.0_centos_linux_amd64.tar.gz'
+          - result.object.sha512 == '518e7c17cf670393045bff4af318e1d35955bfde166e9ceec2b469109252f79043ed133241c4dc96501b6636a1ec5e008ea9ce055d1609865635d4f004d7187b'
+          - result.object.headers['Sensu-Blivet'] == "foo"
+          - result.object.filters == ["entity.system.os == 'linux'", "entity.system.arch == 'amd64'", "entity.system.platform == 'rhel'"]
+          - result.object.metadata.name == 'asset'
```

Comment:
Sweet, this is becoming more and more useful and nicely increases our confidence in modules